templates/_reservations.html

```diff
@@ -5,17 +5,17 @@
 
   <div class="modal-dialog">
     <div class="modal-content left">
-        <form method="get" id="TU_form" target="_blank" action="https://www.opentable.com/restaurant-search.aspx">
```

Comment:
Check if there is any CSS using the id `TU_form`.  If not, delete that from the form since we're not using it.